syntax = "proto3";
package monitor;


message CommandRequest {
    string command = 1;
}



message CommandResponse {
    string timestamp = 1;
    string hostname = 2;
    string metric = 3;
    string value = 4;
    string unit = 5;
}

service MonitorService {
    rpc CommandStream (stream CommandResponse) returns (stream CommandRequest);
}
